Formula/jruby.rb

```diff
@@ -19,6 +19,7 @@ class Jruby < Formula
   end
 
   depends_on "openjdk"
+  conflicts_with "ruby", because: "both install `racc` binaries"
```

Comment:
Does `conflicts_with` do anything when something is already keg-only? Because this should probably be a no-op on macOS where we make `ruby` keg-only.

Reply:
I'm not sure, pinging @Homebrew/brew for the answer.

Reply:
Not sure, try it out. Pretty sure `conflicts_with` prevents them from being installed rather than linked side-by-side.

Reply:
@carlocab @MikeMcQuaid One thing to keep in mind is that even though `ruby` became keg-only in b4bf45228a, existing links to the binaries remain and may still lead to clashes.
Confirmed on my machine, which has had `ruby` installed since many years.

Reply:
How about renaming `racc` for JRuby similar to how we rename:
```rb
      # Prefix a 'j' on some commands to avoid clashing with other rubies
      %w[ast rake rdoc ri testrb].each { |f| mv f, "j#{f}" }
```
As far as I can tell, there isn't anything in Homebrew with `jracc` right now.